src/main/java/com/gmail/goosius/siegewar/SiegeController.java

```diff
@@ -510,36 +511,39 @@ public static void startSiege(Block bannerBlock,
 		SiegeController.saveSiege(siege);
 
 		//Call event
-		Bukkit.getPluginManager().callEvent(new SiegeWarStartEvent(siege, townOfSiegeStarter));
+		Bukkit.getPluginManager().callEvent(new SiegeWarStartEvent(siege, townOfSiegeStarter, getGlobalSiegeStartMessage(siege).defaultLocale()));

```

Comment:
🚧 
- It looks to me that, with this PR "getGlobalSiegeStartMessage()" will be called twice.
- This is not ideal for processing, and also could potentially lead to confusing bugs later.
- To improve this I, suggest the following:
   - Before sending the global message, first generate the required string using the new "get method"
   - Then pass in that string to the send message
   - Then later, pass in the same string to the event